internal/usage/usage.go

```diff
@@ -194,6 +194,16 @@ const (
 	CASFilePath                     = "Path to a PEM file containing one or more CAs for database user authentication."
 	Verbose                         = "If true, returns all child jobs in the response."
 	ClusterID                       = "Unique identifier of the cluster."
+	ReferenceTimeZoneOffset         = "The ISO-8601 timezone offset where the Ops Manager host resides."
+	DailySnapshotRetentionDays      = "Number of days to retain daily snapshots. Ops Manager may return values between 1 and 365, inclusive."
+	ClusterCheckpointIntervalMin    = "Number of minutes between successive cluster checkpoints. Ops Manager may return values of 15, 30, or 60."
+	SnapshotIntervalHours           = "Number of hours between snapshots. Ops Manager may return values of 6, 8, 12, or 24."
+	SnapshotRetentionDays           = "Number of days to keep recent snapshots. Ops Manager may return values between 2 and 5, inclusive."
+	WeeklySnapshotRetentionWeeks    = "Number of weeks to retain weekly snapshots. Ops Manager may return values between 1 and 52, inclusive."
+	PointInTimeWindowHours          = "Number of hours in the past for which a point-in-time snapshot can be created."
+	ReferenceHourOfDay              = "Hour of the day to schedule snapshots using a 24 hour clock. Ops Manager may return values between 0 and 23, inclusive."
+	ReferenceMinuteOfHour           = "Minute of the hour to schedule snapshots. Ops Manager may return values between 0 and 59, inclusive."
+	MonthlySnapshotRetentionMonths  = "Number of months to retain monthly snapshots. Ops Manager may return values between 1 and 36, inclusive."
```

Comment:
since these are all for cloud manager as well may want to say something like, `Valid values:  list of values` we have other examples of `Valid values` here in this file